### PR TITLE
create-method

### PR DESCRIPTION
**Pull request by ajgahan19**

### CODE REVIEW

File: spec/requests/orders_spec.rb [hunk 1]
```diff
@@ -10,5 +10,20 @@
...
+      user = User.where(email: "test@example.com").first_or_create(password: "password", password_confirmation: "password")
+      order_params = { 
+        order:{
+          user_id: 1,
+          product_id: 1,
+      }}
+      post '/orders', params: order_params
+        expect(response).to have_http_status(200)
+        order = order.first
+        expect(order.user_id).to eq 1
```
Comment (elysemontano) [inline spec/requests/orders_spec.rb]: Is this test passing?  Do you need a product to have been created like user is on line 17 for orders?

**Review by elysemontano — DISMISSED** [review]
Left a comment in orders spec file

**Review by ajgahan19 — COMMENTED** [review]
Migrated the product table to add user id 
migrated the products table to include user_id 
changed associations to fix the  rspec testing.

**Review by ajgahan19 — COMMENTED** [review]
Fixed the orders spec test 
Update Associations
Adjusted the seeds to create an order

**Review by cesarcuadro — DISMISSED** [review]

**Review by triciasykes — APPROVED** [review]

**Review by SunkissedQueen — APPROVED** [review]
Approving because I see you have Tricia's seal

**Review by SunkissedQueen — APPROVED** [review]

**Review by SunkissedQueen — APPROVED** [review]

**Pull request merged**